update_flextool.py

```diff
@@ -0,0 +1,43 @@
+import json
+import os
+import subprocess
+
+from migrate_database import migrate_database
+
+
+def update_flextool():
+
+    #completed = subprocess.run(["git","restore", "."])
+    completed = subprocess.run(["git","pull"])
+
+    if completed != 0:
```

Comment:
`subprocess.run()` returns a `CompletedProcess` instance, not exit code as documented [here](https://docs.python.org/3.8/library/subprocess.html#subprocess.CompletedProcess). You should rather test `completed.returncode != 0`